projects/aca-content/src/lib/components/search/search-input/search-input.component.spec.ts

```diff
@@ -239,4 +239,205 @@ describe('SearchInputComponent', () => {
     });
     fixture.detectChanges();
   });
+
+  describe('searchByOption()', () => {
+    let isLibrariesCheckedSpy: jasmine.Spy;
```

Comment:
jasmine.Spy is generic type so please set type of function which you mock in place of <> 
![Uploading Screenshot 2024-11-26 at 14.09.53.png…]()